src/Analyzers/CSharp/Tests/DisambiguateSameVariable/DisambiguateSameVariableTests.cs

```diff
@@ -2,79 +2,78 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CSharp.DisambiguateSameVariable;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Testing;
 using Roslyn.Test.Utilities;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.DisambiguateSameVariable
 {
-    public class DisambiguateSameVariableTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
-    {
-        public DisambiguateSameVariableTests(ITestOutputHelper logger)
-           : base(logger)
-        {
-        }
-
-        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
-            => (null, new CSharpDisambiguateSameVariableCodeFixProvider());
+    using VerifyCS = CSharpCodeFixVerifier<
+        EmptyDiagnosticAnalyzer,
+        CSharpDisambiguateSameVariableCodeFixProvider>;
 
+    public class DisambiguateSameVariableTests
+    {
         [Fact]
         public async Task TestParamToParamWithNoMatch()
         {
-            await TestMissingAsync(
+            var code =
 @"class C
 {
     void M(int a)
     {
-        [|a = a|];
+        {|CS1717:a = a|};
     }
-}");
+}";
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact]
         public async Task TestLocalToLocalWithNoMatch()
         {
-            await TestMissingAsync(
+            var code =
 @"class C
 {
-    void M(int a)
+    void M()
```

Comment:
Please don't change test inputs. Rename test, or add new test with the new input.

Reply:
If you look closely, this is the same test as above, but with different name:
![firefox_MgLlkVAtb6](https://user-images.githubusercontent.com/70431552/204031130-0d52d859-9686-4e45-98ab-12c842cc73f9.png)
This is 100% unintended, so I just "fixed" it by changing it to what it is meant to do according to its name.

Reply:
Sounds good. In the future, just comment the pr to help the reviewer. Thanks!